monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -181,16 +191,13 @@ for f in monitoring/rules/viya/rules-*.yaml; do
   kubectl apply -n $MON_NS -f $f
 done
 
-kubectl get prometheusrule -n $MON_NS v4m-kubernetes-apps 2>/dev/null
-if [ $? == 0 ]; then
-  log_info "Patching KubeHpaMaxedOut rule..."
-  # Fixes the issue of false positives when max replicas == 1
-  kubectl patch prometheusrule --type='json' -n $MON_NS v4m-kubernetes-apps --patch "$(cat monitoring/kube-hpa-alert-patch.json)"
-else
-  log_debug "PrometheusRule $MON_NS/v4m-kubernetes-apps does not exist"
-fi
-
 echo ""
 monitoring/bin/deploy_dashboards.sh
 
 log_notice "Successfully deployed components to the [$MON_NS] namespace"
+if [ "$showPass" == "true" ]; then
+  log_notice ""
+  log_notice "Generated Grafana admin password is: $grafanaPwd"
+  log_notice "Change the password at any time by running (replace pod name and password):"
+  log_notice "kubectl exec -n $MON_NS grafana-pod-name -- bin/grafana-cli admin reset-admin-password myNewPassword"
```

Comment:
I think including the name of the actual/current Grafana pod would be more helpful.  On the logging side, when we've shown commands in this type of message, they can generally be cut-and-pasted immediately to use.  In this case, no one should want to use _myNewPassword_ as their new password, so they have to edit the command anyway but still...  

  I know any pod name shown can become obsolete fairly quickly.  But, if someone has just run the deployment script, it seems unhelpful not to provide a command they can use right away.

Reply:
Yeah, you're right. I'll add some logic to find the pod name that will at least be valid as of the time the deploy script was run.